help/user-guide/isajson-templates.md

```diff
@@ -2,28 +2,38 @@
 title: SEEK User Guide - Experiment Sample Templates
 layout: page
 ---
+# Experiment Sample Templates
 
-# For system administrator
+Experiment Sample Templates act as blueprints to create Sample Types within ISA Studies and ISA Assays and ensure that
+the metadata collected conforms to community standards. The same Experiment Sample Template can be applied multiple times to create Sample Types in different ISA Studies or ISA Assays.
 
-## JSON upload via Server admin panel
-If ISA-JSON compliance feature is enabled, an option appears in the Server admin panel to upload instance-wide Experiment Sample Templates, compliant to ISA-JSON schemas.
+The ISA Study Sources, Samples and ISA Assay Samples tables are Sample Types associated with the ISA Study or ISA Assay and can only be accessed through the ISA Study or ISA Assay interface. The tables can only be created starting from an existing Experiment Sample Template.
 
-The Experiment Sample Templates must be in .json format and comply to the expected specification.
 
-![experiment view](/images/user-guide/isajson-compliance/serveradmin-expsampletemplate-jsonupload.png){:.screenshot}
+An Experiment Sample Template must have the following specifications.
+1. One ISA Level
+   * Study Source
+   * Study Sample
+   * Assay material
+   * Assay data file
+
+2. One Repository name or SEEK Project
+   * Repository or data archive (e.g. EBI databases)
+   * SEEK Project (Project specific templates)
+
+3. Organism: optional free text
 
-## ISA minimal starter template
-When the ISA-JSON compliance feature is enabled in the platform, it is recommended to use the "ISA minimal starter template" as the starting point for creating any new Experiment Sample Template. One ISA minimal starter template is provided for each ISA Level with the feature.
 
-<!--where to find them, specification-->
+## Platform-wide and project specific Experiment Sample Templates
 
-# For project members
```

Comment:
please, keep the distinction "for instance administrator" and "for project members" as main title (1 #).